src/Core/src/Core.csproj

```diff
@@ -21,4 +21,15 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Essentials\src\Essentials.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" PrivateAssets="All" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);RS0041</NoWarn>
```

Comment:
what is this warning?

Reply:
lol not sure i copied from blazor :P

Reply:
@SteveSandersonMS can you tell us?

Reply:
It’s the “oblivious reference types” warning (https://github.com/dotnet/roslyn-analyzers/issues/3793)

I’d recommend removing this suppression from your project and only putting it back if you really need it.